common/lib/dependabot/version.rb

```diff
@@ -1,5 +1,22 @@
 # frozen_string_literal: true
 
 module Dependabot
-  VERSION = "0.214.0"
```

Comment:
I wonder if this was being used somewhere? 

It seems like this `Dependabot::Version` class correlates to the tagged releases of Dependabot and is not the same thing as the package manager version class that we register like in https://github.com/dependabot/dependabot-core/blob/1f5eb050643a1134325cd40e3e14a4ea9055a4b4/bundler/lib/dependabot/bundler/version.rb#L13

Reply:
Yeah, this is just the version of dependabot itself. It's used in the various gemspecs. I moved it to `common/lib/dependabot.rb`. It seemed a better fit since now having a `Dependabot::Version` class but I'm not totally sold on it.